libweb3jsonrpc/Debug.cpp

```diff
@@ -54,6 +53,21 @@ Json::Value Debug::debug_trace(string const& _blockNumberOrHash, int _txIndex)
 			cwarn << diagnostic_information(_e);
 		}
 	}
-	
+	return ret;
+}
+
+Json::Value Debug::debug_storageAt(string const& _blockNumberOrHash, int _txIndex, string const& _address)
```

Comment:
debug_trace also operate on both... do you think operating on the block number could gives bad performance?